tests/vendor/trezor/utils.py

```diff
@@ -1,22 +1,40 @@
+import os
 import socket
 from fido2.ctap import STATUS
 
 from trezorlib import debuglink
 from trezorlib.debuglink import TrezorClientDebugLink
 from trezorlib.device import wipe as wipe_device
-from trezorlib.transport import enumerate_devices
+from trezorlib.transport import enumerate_devices, get_transport
 
 
-def load_client():
-    devices = enumerate_devices()
-    for device in devices:
+def get_device():
+    path = os.environ.get("TREZOR_PATH")
+    interact = int(os.environ.get("INTERACT", 0))
```

Comment:
Do we need/want `INTERACT` here?
It disables debuglink decisions so that you can click on the device/emulator manually.

Reply:
Probably not, but then why do we have it in the device tests?

Reply:
It's a debugging aid. You can run a particular test anytime with `INTERACT=1`, without rebuilding the emulator, see the screens one by one and walk through the UI flow.
I'm not sure how useful this would be for FIDO tests, that is why i'm asking. Isn't there an option (leaving out `--sim`?) to accomplish the same?